src/Packs.sol

```diff
@@ -703,6 +712,7 @@ contract Packs is
     /// @dev Only callable by admin role
     /// @dev Emits a MaxPackPriceUpdated event
     function setMaxPackPrice(uint256 maxPackPrice_) external onlyRole(DEFAULT_ADMIN_ROLE) {
+        if (maxPackPrice_ == 0) revert InvalidPackPrice();
```

Comment:
You get this for free with,

`if (maxPackPrice_ < minPackPrice) revert InvalidPackPrice();`